execution-environment/execution-environment.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
what do you think about adding a README here as an instruction for those who would like to run it by themselfs?